shell/server/pre-meteor.js

```diff
@@ -90,7 +96,15 @@ Meteor.startup(function () {
       publicIdPromise = Promise.resolve(id);
     } else {
       // Not a wildcard host. Perhaps it is a custom host.
-      publicIdPromise = lookupPublicIdFromDns(hostname);
+
+      // Skip DNS TXT record resolutions for the major pseudo-TLDs, and avoid
+      // returning any result which might indicate that there's actually a
+      // server listening here.
+      if (hasPseudoTLD(hostname)) {
+        return;
```

Comment:
I'm not sure if this is correct here… would Meteor prefer getting a new `Promise` that eventually returns null/undefined?